extensions/amp-youtube/validator-amp-youtube.protoascii

```diff
@@ -33,9 +33,14 @@ tags: {  # <amp-youtube>
     name: "credentials"
     value_regex_casei: "(include|omit)"
   }
+  attrs: {
+    name: "data-live-channelid"
```

Comment:
please add a few tests to validator-amp-youtube.html (and .out) to cover success and failure of this new rule.